test/tests/meshgenerators/distributed_rectilinear_mesh_generator/tests

```diff
@@ -47,4 +47,23 @@
     requirement = 'MOOSE shall be able to generate 3D HEX8 mesh in parallel using hierarch partitioner'
   [../]
 
+  [./drmg_3d_scomm_out]
+    type = 'Exodiff'
+    input = 'distributed_rectilinear_mesh_generator.i'
+    exodiff = 'drmg_3d_scomm_out.e'
+    min_parallel = 3
+    issues = '#15464'
+    cli_args = 'Mesh/gmg/num_cores_for_partition=2 Mesh/gmg/dim=3 Mesh/gmg/nx=20 Mesh/gmg/ny=20 Mesh/gmg/nz=20 Outputs/file_base=drmg_3d_scomm_out Outputs/hide="pid npid" '
+    requirement = 'MOOSE shall be able to generate 3D HEX8 mesh in parallel using a small number of cores for partition.'
```

Comment:
@permcody taught me to try to avoid implementation details in requirements. Maybe instead of HEX8, say "first-order hexahedral". Also "for partition" -> "to partition" or "for partition" -> "for the mesh partitioner"

Reply:
Done